test/models/todo_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```

Comment:
This and everything else in the test/ folder can be removed, since we use Rspec here.